### PR TITLE
Drop the polyfill from the element.classList doc; add link

### LINKED ISSUE #2224: Issue with "Element.classList": Polyfill no longer working on IE 11
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

#### What information was incorrect, unhelpful, or incomplete?

I am experiencing a problem with `Element.classList.toggle`'s polyfill not working. The second argument, `force`, is ignored on IE 11. It seems like the polyfill may not be applied in this situation. I believe the example has worked in the past.

This can also be reproduce by grabbing the most up-to-date polyfill snippet from MDN and running a basic example.

#### Specific section or headline?

`#polyfill`

#### What did you expect to see?

The `force` argument to work in IE 11.

#### Did you test this? If so, how?

Build a test and open in a modern browser and IE 11. You would expect the element to remain red, but the class is instead removed (i.e. the element has no background.)

```html
<style>
  .red {
    background: red;
  }
</style>

<div class="red">
  Red
</div>

<script>
  /**
   * [...]
   * Polyfill from https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#polyfill
   */
  document.querySelector(".red").classList.toggle("red", true);
</script>
```

Related: https://github.com/msn0/mdn-polyfills/issues/57

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/element/classlist`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/api/element/classlist/index.html
* Last commit: https://github.com/mdn/content/commit/068e0f6bfb252775eb9835632c3b0083259c9fcc
* Document last modified: 2021-01-25T04:06:51.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
MDN policy is to no longer maintain polyfill code in MDN articles but to instead link to polyfill code at external sites, if such exists. In this case, Eli Grey's classList.js is a quality element.classList implementation that we can confidently link to.

Fixes https://github.com/mdn/content/issues/2224

### CODE REVIEW

**Comment by peterbe** [comment]
One layman comment; a user might use the Cmd-f to search for the word "polyfill" within the text, and find nothing. 
You called it:

> (a cross-browser JavaScript shim that fully implements element.classList)

could that be simplified to "a cross-browser JavaScript polyfill that fully implements element.classList"?

File: files/en-us/web/api/element/classlist/index.html [hunk 1]
```diff
@@ -277,4 +90,5 @@ <h2 id="See_also">See also</h2>
 <ul>
     <li>{{domxref("element.className")}}</li>
     <li>{{domxref("DOMTokenList")}}</li>
+    <li><a href="https://github.com/eligrey/classList.js"><code>classList.js</code></a> (a cross-browser JavaScript shim that fully implements <code>element.classList</code>)</li>
```
Comment (sideshowbarker) [inline files/en-us/web/api/element/classlist/index.html:93]:
```suggestion
    <li><a href="https://github.com/eligrey/classList.js"><code>classList.js</code></a> (a cross-browser JavaScript polyfill that fully implements <code>element.classList</code>)</li>
```

**Comment by sideshowbarker** [comment]
> could that be simplified to "a cross-browser JavaScript polyfill that fully implements element.classList"?

Yup, thanks — changed it to that

**Review by wbamberg — APPROVED** [review]
Very nice.

**Pull request merged**